posts/2024-05-01-gsoc-2024-selected-projects.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Announcing Google Summer of Code 2024 selected projects"
+author: Jakub Beránek, Jack Huey and Paul Lenz
+---
+
+The Rust Project is [participating][gsoc blog post] in [Google Summer of Code (GSoC) 2024][gsoc], a global program organized by Google which is designed to bring new contributors to the world of open-source.
+
+In February, we published a list of [GSoC project ideas][project idea list], and started discussing these projects with potential GSoC applicants on our [Zulip][zulip gsoc]. We were pleasantly surprised by the amount of people that wanted to participate in these projects and that led many fruitful discussions with members of various Rust teams. Some of them have even immediately begun contributing to various repositories of the Rust Project, even before GSoC officially started!
+
+After the initial discussions, GSoC applicants prepared and submitted their project proposals. We have received 65 (!) proposals in total. We are happy to see that there was so much interest, given that this is the first time the Rust Project participates in GSoC.
+
+A team of mentors primarily composed of Rust Project contributors then thoroughly examined the submitted proposals. GSoC required us to produce a ranked list of the best proposals, which was a challenging task in itself, since Rust is a big project with many priorities! We went through many rounds of discussions, and had to consider many factors, such as prior conversations with the given applicant, the quality and scope of their proposal, the importance of the proposed project for the Rust Project and its wider community, but also the availability of mentors, who are often volunteers and thus have limited time available for mentoring.
```

Comment:
```suggestion
A team of mentors primarily composed of Rust Project contributors then thoroughly examined the submitted proposals. GSoC required us to produce a ranked list of the best proposals, which was a challenging task in itself since Rust is a big project with many priorities! We went through many rounds of discussions and had to consider many factors, such as prior conversations with the given applicant, the quality and scope of their proposal, the importance of the proposed project for the Rust Project and its wider community, but also the availability of mentors, who are often volunteers and thus have limited time available for mentoring.
```